src/Routes.php

```diff
@@ -161,9 +163,21 @@ public function endpoints(): array
             return $endpoints;
         }
 
-        return array_filter($endpoints, function($endpoint) {
-            return in_array($endpoint['type'], $this->types);
-        });
+        if (isset($this->types['only']) and count($this->types['only'])) {
+            $only = $this->types['only'];
+            $endpoints = array_filter($endpoints, function($endpoint) use ($only) {
+                return in_array($endpoint['type'], $only);
```

Comment:
I think you have access to `$this` in the callback.. no?

```php
return in_array($endpoint['type'], $this->types['only']);
```

Reply:
I already did `$only = $this->types['only'];` on line number 167. so I think this one is good.